README.md

```diff
@@ -109,6 +109,8 @@ Can be a [glob](https://github.com/isaacs/node-glob#glob-primer) or and `Array`
 | `name`   | The name of the downloadable file on the GitHub release.                                                 | File name extracted from the `path`. |
 | `label`  | Short description of the file displayed on the GitHub release.                                           | -                                    |
 
+`name` and `label` may contain the placeholder `%RELEASE%`, which will be replaced with the full tag of the release.
```

Comment:
I'm not sure what you mean with `full tag`? What would a partial tag be?

This would be more cleare?

```suggestion
 `name` and `label` may contain the placeholder `%RELEASE%`, which will be replaced with the tag name of the release.
```